crates/autopilot/src/run_loop.rs

```diff
@@ -510,11 +512,13 @@ impl RunLoop {
 
         // Shuffle so that sorting randomly splits ties.
         solutions.shuffle(&mut rand::thread_rng());
-        solutions.sort_unstable_by_key(|participant| participant.solution.score().get().0);
+        solutions.sort_unstable_by_key(|participant| {
+            std::cmp::Reverse(participant.solution.score().get().0)
+        });
```

Comment:
At least the `get_status` API endpoint and FE logic depend on the sorting. cc @alfetopito 
https://github.com/cowprotocol/services/blob/649e32ec15376f68541dc12642b19baca7b72846/crates/orderbook/src/orderbook.rs#L454-L458

Reply:
Ah good one, I introduces this bug with the latest commit.

Reply:
Fixed